plutarch-test/src/Plutarch/Test.hs

```diff
@@ -170,26 +172,38 @@ golden pg p =
 -}
 goldens :: PlutarchGolden -> [(String, ClosedTerm a)] -> Spec
 goldens pg ps = do
-  testAncestors <- fmap (drop 1 . reverse) $ getTestDescriptionPath
-  let name = T.unpack $ T.intercalate "." testAncestors
+  name <- getGoldenFilePrefix
   describe "golden" $ do
     -- Golden test for UPLC
     when (hasPrintTermGolden pg) $ do
       it "uplc" $
-        pureGoldenTextFile ("goldens" </> name <> ".uplc.golden") $
+        pureGoldenTextFile (goldenFilePath "goldens" name "uplc") $
```

Comment:
Is this just a general clean-up?

Reply:
it's used as a helper for the new tests, too